nuxt.config.js

```diff
@@ -41,6 +41,7 @@ module.exports = {
     }
   ],
   env: {
+    mainnetURL: process.env.NUXT_APP_NODE_URL || 'https://mainnet.aeternity.io/v3',
     middlewareURL: process.env.NUXT_APP_NODE_URL || 'https://mainnet.aeternity.io/mdw',
```

Comment:
this cannot be the same ENV variable like the mdw url and I would rename it anyway like this:

```suggestion
    nodeURL: process.env.NUXT_APP_NODE_URL || 'https://mainnet.aeternity.io/v3',
    middlewareURL: process.env.NUXT_APP_MDW_URL || 'https://mainnet.aeternity.io/mdw',
```

we should call it `nodeURL` as it can also be testnet.